lib/vtls/cipher_suite.c

```diff
@@ -165,12 +165,12 @@ enum {
 
 #define CS_ZIP_IDX(a, b, c, d, e, f, g, h)    \
 {                                             \
-  (uint8_t) ((a) << 2 | ((b) & 0x3F) >> 4),   \
-  (uint8_t) ((b) << 4 | ((c) & 0x3F) >> 2),   \
-  (uint8_t) ((c) << 6 | ((d) & 0x3F)),        \
-  (uint8_t) ((e) << 2 | ((f) & 0x3F) >> 4),   \
-  (uint8_t) ((f) << 4 | ((g) & 0x3F) >> 2),   \
-  (uint8_t) ((g) << 6 | ((h) & 0x3F))         \
+  (uint8_t) (((a) << 2) & 0xff | ((b) & 0x3F) >> 4),   \
+  (uint8_t) (((b) << 4) & 0xff | ((c) & 0x3F) >> 2),   \
+  (uint8_t) (((c) << 6) & 0xff | ((d) & 0x3F)),        \
+  (uint8_t) (((e) << 2) & 0xff | ((f) & 0x3F) >> 4),   \
+  (uint8_t) (((f) << 4) & 0xff | ((g) & 0x3F) >> 2),   \
+  (uint8_t) (((g) << 6) & 0xff | ((h) & 0x3F))         \
```

Comment:
Tiny nit, for consistency, an uppercase `0xFF` fits better here.

Reply:
Done.